Agile-Squad-Definition-Discovery.md

```diff
@@ -8,7 +8,7 @@ Source Document: https://github.com/alexwitherspoon/Agile-Squad-Organization-Mod
 
 **Why Defining the Squad Matters**
 
-For an Agile Squad to best function, they need to be able to define what they are authoritative for, the dependencies of that Squad, and describe the goals of that Squad. This helps Squads define areas of autonomy, and declare clear dependencies between Squads. 
+For an Agile Squad to best function, they need to be able to define what they control, what they depend on, and what their goals are. This helps Squads define areas of autonomy, and declare clear dependencies between Squads.
```

Comment:
This is good - I used the word authoritative because it plays into the RACI language- https://en.wikipedia.org/wiki/Responsibility_assignment_matrix

But I like how your wording sounds. I'll go into that depth in a later article/section and accept the changes you made for human clarity sake.